symfony/messenger/4.3/post-install.txt

```diff
@@ -3,13 +3,13 @@
 <bg=blue;fg=white>              </>
 
   * You're ready to use the Messenger component. You can define your own message buses
-    or start using the default one right now by injecting the <info>messenger.bus.default</info> service
-    or typehinting <info>Symfony\Component\Messenger\MessageBusInterface</info> in your code.
+    or start using the default one right now by injecting the <info>message_bus</info> service
+    or type-hinting <info>Symfony\Component\Messenger\MessageBusInterface</info> in your code.
 
   * To send messages to a transport and handle them asynchronously:
```

Comment:
This seems to be missing an important step to start the worker.  Otherwise they don't get handled as the description says.